build.go

```diff
@@ -315,6 +317,20 @@ func (c *Client) Build(ctx context.Context, opts BuildOptions) error {
 		}
 	}
 
+	IsSuggestedBuilderFunc := func(b string) bool {
+		for _, suggestedBuilder := range builder.SuggestedBuilders {
+			if b == suggestedBuilder.Image {
+				return true
+			}
+		}
+		return false
+	}
```

Comment:
nit: Since this `func` doesn't depend on anything within the `Build` function, can you define it above?